lib/raygun.breadcrumbs.express.ts

```diff
@@ -0,0 +1,34 @@
+import type { Request } from "express";
+import type { Breadcrumb } from "./types";
+import { getBreadcrumbs } from "./raygun.breadcrumbs";
+
+const debug = require("debug")("raygun");
+
+/**
+ * Parses an ExpressJS Request and adds it to the breadcrumbs store
+ * @param request
+ */
+export function addRequestBreadcrumb(request: Request) {
+  const crumbs = getBreadcrumbs();
+
+  if (!crumbs) {
+    debug(
+      "[raygun.breadcrumbs.express.ts] Add request breadcrumb skip, no store!",
+    );
+    return;
+  }
+
+  const internalCrumb: Breadcrumb = {
```

Comment:
Perhaps we need to add additional code comments around the internal breadcrumbs approach to mention what is user facing and internal